src/frogfs.c

```diff
@@ -137,7 +137,7 @@ static const void *find_object(frogfs_fs_t *fs, const char *path)
     LOGV(__func__, "%s", path);
 
     uint32_t hash = djb2_hash(path);
-    LOGV(__func__, "hash %08x", hash);
+    LOGV(__func__, "hash %08"PRIx32, hash);
```

Comment:
What does this do?

Reply:
Since last versions of gcc in esp-idf package requeue to specify exactly type of variable in printf format string this line generate error on compilation time to change x to lx (uint32_t is unsigned long type so l-modifier is requeued). To make it change more readable exists macroses for printf formatting: PRI<type><size>, so PRIx32 is equivalent to %lx.